packages/manager/apps/ips/public/translations/ips/Messages_fr_FR.json

```diff
@@ -1,4 +1,5 @@
 {
   "breadcrumb_root_label": "Toutes les IPs",
-  "order": "Commander une IP"
+  "order": "Commander une IP",
+  "free_price": "Gratuit"
```

Comment:
Can be placed in common_translation module ?

Reply:
I think it should be in the price component if other apps need it
But it's a variant of the current label "included" when the price is 0